src/test/java/example/Example.java

```diff
@@ -86,5 +94,21 @@ public void initialize(Bootstrap<ExampleConfiguration> bootstrap) {
         public void run(ExampleConfiguration configuration, Environment environment) throws Exception {
             environment.jersey().register(new ExampleResource());
         }
+
+        @VisibleForTesting
+        WebSecurityBundle getWebSecurityBundle() {
+            return this.webSecurityBundle;
+        }
+    }
+
+    /**
+     * Example REST Application.
+     */
+    public static final class ExampleRestApplication extends ExampleWebApplication {
```

Comment:
can this be its own separate app instead of extending the "web" app for  "rest" app?